core/modules/layout/css/grid-float.css

```diff
@@ -950,3 +960,63 @@
     margin-left: 91.666667%;
   }
 }
+
+/**
+ * Override flexbox values in case they're set
+ */
+.col-xs-first,
+.col-xs-last,
+.col-sm-first,
+.col-sm-last,
+.col-md-first,
+.col-md-last,
+.col-lg-first,
+.col-lg-last,
+.col-xl-first,
+.col-xl-last {
+  -webkit-box-ordinal-group: unset;
+  -webkit-order: unset;
+  -ms-flex-order:unset;
```

Comment:
`-ms-flex-order` apparently doesn't support `unset`, but you could put it back to the default value of `0`.